### PR TITLE
EIP-1559 V2: Use default gas preferences of user for all transactions

### LINKED ISSUE #12587: Use default gas preferences of user for all transactions
As part of code changes for issue https://github.com/MetaMask/metamask-extension/issues/12447 default gas settings are saved for user.

If a user has these defaults configured we need to ensure that the defaults are always used as a new transaction comes into metamask, user can then change the gas selection is user does not wants to use default.

We need to update initialisation of `estimateToUse` here: https://github.com/MetaMask/metamask-extension/blob/develop/ui/hooks/gasFeeInput/useGasFeeInputs.js#L78

and also `isUsingDappSuggestedGasFees` here: https://github.com/MetaMask/metamask-extension/pull/12567/files#diff-34d0d466a71686f66f2b6fd8cad9426791bb1518008487a18497d2c018234394R100

### PR DESCRIPTION
**Pull request by NiranjanaBinoy**
Fixes: #12587 

Explanation:
- Removing the multiplier from the max base fee and related changes
- When the default values for advancedGasFee are saved by selecting the checkbox in the <AdvancedGasFeePopover/> the Advanced gas settings a.k.a custom gas settings have to be selected by default for all future transactions.

https://user-images.githubusercontent.com/43930900/148617833-04aa8cfa-4995-4bb5-8598-84876d964277.mov

### CODE REVIEW

**Review by jpuri — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -1293,6 +1284,9 @@
   "grantedToWithColon": {
     "message": "Granted to:"
   },
+  "gweiInParanthesis": {
```
Comment (mcmire) [inline app/_locales/en/messages.json:1287]: typo: `gweiInParanthesis` -> `gweiInParentheses`

**Review by mcmire — COMMENTED** [review]
One little thing, but this looks good otherwise.

File: app/_locales/en/messages.json (hunk 1 above)
Comment (NiranjanaBinoy) [reply app/_locales/en/messages.json:1287]: 😅👍

Comment (NiranjanaBinoy) [reply app/_locales/en/messages.json]: done 😅

**Review by mcmire — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -1293,6 +1284,9 @@
   "grantedToWithColon": {
     "message": "Granted to:"
   },
+  "gweiInParentheses": {
+    "message": "(GWEI)"
+  },
```
Comment (jpuri) [inline app/_locales/en/messages.json:1289]: I think we can leave out parentheses from internationalised message

**Review by jpuri — DISMISSED** [review]

**Comment by jpuri** [comment]
@NiranjanaBinoy : a fix may be needed here also, plz check https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/edit-gas-fee-popover/edit-gas-item/useGasItemFeeDetails.js#L62

**Review by jpuri — APPROVED** [review]

**Comment by jpuri** [comment]
Will be useful to have some unit test cases added for advance gas configurations.

**Review by mcmire — APPROVED** [review]

**Pull request merged**